<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickCurveDataModeSetDialog</class>
 <widget class="QDialog" name="PickCurveDataModeSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick Curve Data Mode Set</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_range">
     <property name="title">
      <string>select export datas range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_viewRange">
        <property name="text">
         <string>export current select region datas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_allRange">
        <property name="text">
         <string>export all datas</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_type">
     <property name="title">
      <string>choose export data type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_xOnly">
        <property name="text">
         <string>export x value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_yOnly">
        <property name="text">
         <string>export y value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_point">
        <property name="text">
         <string>export (x,y) value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PickCurveDataModeSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
